# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1286638380 14400
#      Sat Oct 09 11:33:00 2010 -0400
# Node ID 8b9aeb20c03c55aeb873a90b79af33b1d7342cf0
# Parent  5677f3f7b5fac3042528bea7d13fd97922d14ff7
__fltk_ginput__.m: use || instead of | in IF condition

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-10-09  John W. Eaton  <jwe@octave.org>
+
+	* plot/__fltk_ginput__.m: Use || instead of | in IF condition.
+
 2010-10-08  Ben Abbott <bpabbott@mac.com>
 
 	* image/image.m, plot/__go_draw_axes__.m: Do not store flipped image
 	property values when x/ydata descends.
 
 2010-10-07  John W. Eaton  <jwe@octave.org>
 
 	* signal/periodogram.m: Fix parse errors.
diff --git a/scripts/plot/__fltk_ginput__.m b/scripts/plot/__fltk_ginput__.m
--- a/scripts/plot/__fltk_ginput__.m
+++ b/scripts/plot/__fltk_ginput__.m
@@ -44,17 +44,17 @@ function [x, y, button] = __fltk_ginput_
     
     while (true)
       __fltk_redraw__ ();
       
       ## release CPU
       sleep (0.01);
 
       [x, y, n0] = ginput_aggregator (-1, 0, 0);
-      if (n0 == n | n0 < 0)
+      if (n0 == n || n0 < 0)
         break;
       endif
     endwhile
     
     ## FIXME -- got to get the buttons somehow
     button = ones (size (x));
   unwind_protect_cleanup
     set (f, "windowbuttondownfcn", orig_windowbuttondownfcn);
